src/main/python/calibration/generate_rides_with_google_maps.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```

Comment:
How can we be sure that use has Python3 and all the needed dependencies? Should we have `requirement.txt` for `pip` and make sure users install it before running Beam?